docs/ref/middleware.txt

```diff
@@ -61,6 +61,19 @@ Adds a few conveniences for perfectionists:
   indexer would treat them as separate URLs -- so it's best practice to
   normalize URLs.
 
+  If necessary, individual views may be excluded from the ``APPEND_SLASH``
+  behavior by setting a ``_should_append_slash`` attribute on the view itself::
+
+    def sensitive_fbv(request, *args, **kwargs):
+        """View to be excluded from APPEND_SLASH."""
+        return HttpResponse()
+
+
+    sensitive_fbv._should_append_slash = False
```

Comment:
If we're documenting this as public API, we should probably drop the "private" underscore prefix...

Reply:
Yep. @felixxm already suggested that. Also we'll probably have a decorator, that'll be a bit nicer.

Reply:
Cool. I finally got around to decorators for the admin in #13532. They do make things much nicer!

Reply:
Yes, they look quite good. I should keep up with the latest developments! 😀